src/features/TransactionTable/TransactionTable.utils.ts

```
@@ -0,0 +1,21 @@
+export const formatCurrency = (amount: number, symbol: string = '$') =>
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)